biztalk/core/azure-blob-adapter.md

```diff
@@ -0,0 +1,153 @@
+---
+title: "Use the Azure Blob adapter | Microsoft Docs"
+description: Send and receive messages using the Azure Blob adapter in BizTalk Server
+author: "Elvis-Shi"
+ms.author: "elsh"
+manager: "dougeby"
+ms.date: "12/23/2019"
+ms.topic: conceptual
+ms.prod: biztalk-server
+
+# optional metadata
+
+#ROBOTS:
+
+ms.reviewer: 
+ms.suite:
+ms.tgt_pltfrm:
+ms.assetid: 
+ms.custom: "biztalk-2020"
+
+---
+
+# Azure Blob storage adapter in BizTalk
+
+## Overview
+**Starting with BizTalk Server 2020**, you can send and receive messages between BizTalk Server and Azure Blob storage. 
+
+Azure Blob storage is Microsoft's object storage solution for the cloud, which is optimized for storing massive amounts of unstructured data. [What is Azure Blob storage?](https://docs.microsoft.com/azure/storage/blobs/storage-blobs-overview) provides more details.
+
+## Prerequisites
+
+* Create an [Azure blob storage account with a container](https://docs.microsoft.com/azure/storage/common/storage-create-storage-account)
+
+## Send messages to Azure Blob storage
+
+1.  In the BizTalk Server Administration console, right-click **Send Ports**, select **New**, and select **Static One-way send port**.
+
+    [Create a Send Port](../core/how-to-create-a-send-port2.md) provides some guidance.
+
+2. Enter a **Name**. In **Transport**, set the **Type** to **AzureBlobStorage**, and click **Configure** button. 
+
+3. Configure the **Azure Account** properties: 
+
+    |Use this|To do this|  
+    |---|---|  
+    | **Sign-in** | Sign into your Azure account |
+    | **Subscription** | Select the subscription that has your Azure storage account |
+    | **Resource Group** | Select your resource group that has your Azure storage account |
+
+4. Configure the **General** properties: 
+
+    |Use this|To do this|  
+    |---|---|  
+    | **Storage Authentication** | Select an authentication method. <ul><li>Typically, it's recommended to use a Shared Access Signature, which is also selected by default. You can input the Shared Access Signature connection string in the **Connection string** field to provide the authentication.</li> <li>If you are using Access keys as authentication method, a collection of storage account will be populated up in **Account** drop-down list. After you select the storage account, **Connection string** field will be automatically populated with your primary access key, which also known as **key1**. </li></ul><br />The following links are good resources to help you decide which authentication method is right for your scenario:<br/><br/>[About Azure storage accounts](https://docs.microsoft.com/azure/storage/common/storage-create-storage-account)<br/>[Using shared access signatures (SAS)](https://docs.microsoft.com/azure/storage/common/storage-dotnet-shared-access-signature-part-1) |
```

Comment:
This is somewhat confusing without a background in Storage.

Reply:
I understand, this is a common scenario, everyone document actually have some Prerequisites for reader. The whole article structure i followed event hub adapter, unless you are proposing somehting else.